lib/checks/aria/aria-hidden-focus.js

```diff
@@ -0,0 +1,15 @@
+const { dom } = axe.commons;
+
+let elements = [node];
+
+if (node.children.length) {
+	const children = Array.prototype.slice.call(node.querySelectorAll('*'));
```

Comment:
This should query across Shadow DOM boundaries, as aria-hidden does cascade in there. Demo to check: https://codepen.io/marcysutton/pen/bQGpEy?editors=1010